File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_channel_account_query_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusChannelAccountQueryForm:
    """Query form with optional filters"""
    id: int = None
    created_at_from: str = None
    created_at_to: str = None
    channel: str
    status: str
    resource_type: str = None
    resource_types: List[str] = None

File: sdkwork-backend-sdk-go/types/plus_channel_account_query_form.go
package types

// Query form with optional filters
type PlusChannelAccountQueryForm struct {
	Id int `json:"id"`
	CreatedAtFrom string `json:"createdAtFrom"`
	CreatedAtTo string `json:"createdAtTo"`
	Channel string `json:"channel"`
	Status string `json:"status"`
	ResourceType string `json:"resourceType"`
	ResourceTypes []string `json:"resourceTypes"`
}
